helpers/handlebars/helpers/index.js

```diff
@@ -127,6 +127,12 @@ module.exports = {
 		}
 		return options.fn(this);
 	},
+	env: (envVariable, options) => {
+		if (process.env[envVariable] !== undefined) {
+			return process.env[envVariable];
```

Comment:
@CeEv @tofixx  Any performance concerns here regarding process.env?

Reply:
let's  replace that with commons.Configuration asap 
but for now just do a lodash deepclone once the file gets loaded from process.env